src/blocks/remote-data-no-results/index.ts

```diff
@@ -1,11 +1,23 @@
 import { registerBlockType } from '@wordpress/blocks';
+import { border, caution } from '@wordpress/icons';
 
 import metadata from './block.json';
 import { Edit } from './edit';
 import { Save } from './save';
 import './style.scss';
 
+// @ts-expect-error - WordPress registerBlockType type definition does not include variations even though this is the right way to do it.
```

Comment:
The type definitions for this method aren't correct which is why typescript throws an error here. We can provide the variations, and that's needed as without that the icon will not dynamically change between border and caution.

A follow up here would be to figure out how we can update those type definitions once the pause is lifted.

Reply:
Added this in, hopefully I've done it correctly